hw/syn/data/dc.hjson

```diff
@@ -0,0 +1,15 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  tool_srcs: ["{proj_root}/hw/foundry/syn/{tool}/setup.tcl"
+              "{proj_root}/hw/syn/tools/{tool}/start-dc.sh"
+              "{proj_root}/hw/syn/tools/{tool}/run-syn.tcl"
+              "{proj_root}/hw/syn/tools/{tool}/constraints.sdc"]
+
+  // we rely on fusesoc to run lint for us
```

Comment:
that comment looks wrong

Reply:
yeah this was copy pasted from the lint flow... removed.